.github/workflows/post-merge.yaml

```diff
@@ -0,0 +1,41 @@
+name: Post Merge
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  generate:
+    name: Update versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Commit & Push
+        shell: bash
+        run: |
```

Comment:
This was taken from OPA, https://github.com/open-policy-agent/opa/blob/main/.github/workflows/post-merge.yaml#L21-L46